theme/errorList.less

```diff
@@ -20,9 +21,35 @@
         height: 200px;
         border-top: 1px #e5e5e5 solid;
         padding: 1em;
+
+        h4 {
+            margin-bottom: 8px;
```

Comment:
To keep consistent with the rest of the spacing, maybe 0.5em here instead of 8px? Up to you ~